servicemap/rest_client.py

```diff
@@ -0,0 +1,45 @@
+import requests
+
+
+class ServicemapApiClient:
+
+    CONNECTION_TIMEOUT = 20
+    DATA_SOURCE = "tprek"
+
+    def __init__(self, config) -> None:
+        self.root = config["ROOT"]
+
+    def list_helsinki_schools_and_kindergartens(self, filter_list=None):
+        """
+        List of names and ids' of schools, colleges and kindergardens.
+        Query:
+        https://api.hel.fi/servicemap/v2/unit/?municipality=helsinki&only=name&service_node=2118,868,2179,1088,1257,1097
+        """
+        service_nodes = [2118, 868, 2179, 1088, 1257, 1097]
```

Comment:
I wonder if it would make sense to have this in settings e.g. in the `SERVICEMAP_API_CONFIG`

Reply:
I thought so first too... It could, but the service nodes should be VERY static and should only affect in this 1 specific API call. In that sense, it 's nothing but more things to configure.

Reply:
I will add comments about what they mean next to those service node codes.